cage.c

```diff
@@ -93,8 +95,11 @@ static bool
 parse_args(struct cg_server *server, int argc, char *argv[])
 {
 	int c;
-	while ((c = getopt(argc, argv, "hD")) != -1) {
+	while ((c = getopt(argc, argv, "dDh")) != -1) {
 		switch (c) {
+		case 'd':
+			server->xdg_decoration = true;
```

Comment:
This conflicts the description in the help message (`Don't draw client side decorations`). I suggest declaring `xdg_decoration` as `true` and setting it to false if `-d` is passed.

Reply:
It doesn't conflict. Without xdg decorations, clients draw their CSDs as normal. Only when xdg decorations are enabled within Cage, does Cage negotiate with client that it itself will draw server side decorations. Hence, with xdg decoration *on*, server side decorations will be *on* and CSDs will be *off* (if the client supports the protocol, that is).